File: problemas/strings/2174/2174.cpp
#include <iostream>
#include <set>

using namespace std;

int main(){
    int N;
    string S;
    set<string> pomekons;

    cin >> N;

    for(int i = 0; i < N; ++i){
        cin >> S;
        
        pomekons.insert(S);
    }

    cout << "Falta(m) " << 151 - pomekons.size() << " pomekon(s)." << endl;

    return 0;
}

File: problemas/strings/2174/2174.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.HashSet;

public class Main {
    public static void main(String[] args) throws IOException {
        InputStreamReader ir = new InputStreamReader(System.in);
        BufferedReader in = new BufferedReader(ir);

        int N = Integer.parseInt(in.readLine());
        HashSet<String> pomekons = new HashSet<String>();
        for(int i = 0; i < N; ++i){
            String pomekon = in.readLine().trim();
            pomekons.add(pomekon);
        }

        System.out.printf("Falta(m) %d pomekon(s).\n", 151 - pomekons.size());
    }
}